contents/docs/product-analytics/autocapture.mdx

```diff
@@ -16,7 +16,9 @@ PostHog can capture frontend events automatically using autocapture. This captur
 
 This means you don't need to manually add tracking for individual components, links, buttons, divs, spans, or other parts of your product. Autocapture also provides data for [heatmaps](/docs/toolbar/heatmaps).
 
-Autocapture is available for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), and [React Native](/docs/libraries/react-native) SDKs and is **enabled by default**.
+Autocapture is available by for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), and [React Native](/docs/libraries/react-native) SDKs and is **enabled by default**. 
+
+Autocapture is also available for our [iOS SDK](/docs/libraries/ios), but it is **disabled by default** as an experimental feature.
```

Comment:
its disabled by default (afaik) in all SDKs due to PII risk, so no need to mention that.
I'd just `, and iOS` with the link and that's good.

Reply:
see this comment https://github.com/PostHog/posthog.com/pull/10085#discussion_r1876083555